CHANGELOG.md

```diff
@@ -35,23 +35,29 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 # Changelog
 
-## [v2.5.0] - TBD
+## [v2.6.0] - 10-21-2022
 
 ### Client Breaking Changes
 * (app) [\#467](https://github.com/shentufoundation/shentu/pull/467) Change binary name to `shentud` from `certik`
 ### API Breaking Changes
 * (app) [\#467](https://github.com/shentufoundation/shentu/pull/467) Change package path to "github.com/shentufoundation/shentu" from "github.com/certikfoundation/shentu"
 ### State Machine Breaking Changes
-* (app) [\#390](https://github.com/shentufoundation/shentu/pull/390) Bump Cosmos SDK to v0.45.4
+* (app) [\#473](https://github.com/shentufoundation/shentu/pull/473) Store migration for mixcoins removal
+* * (app) [\#390](https://github.com/shentufoundation/shentu/pull/390) Bump Cosmos SDK to v0.45.4
```

Comment:
the Cosmos SDK version should be v0.45.9?

Reply:
yes, should be v0.45.9